modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -23,3 +23,42 @@ As a cluster administrator, you can enable IPsec encryption after cluster instal
 $ oc patch networks.operator.openshift.io cluster --type=merge \
 -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipsecConfig":{ }}}}}'
 ----
+
+.Verification
+
+. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app=ovnkube-master -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   READY   STATUS    RESTARTS   AGE
+ovnkube-master-fvtnh   6/6     Running   0          122m
+ovnkube-master-hsgmm   6/6     Running   0          122m
+ovnkube-master-qcmdc   6/6     Running   0          122m
+----
+
+. Verify that IPsec is enabled on your cluster:
++
+[source,terminal]
+----
+$ oc -n openshift-ovn-kubernetes rsh ovnkube-master-<XXXXX> \
+  ovn-nbctl --no-leader-only get nb_global . ipsec
+----
++
+--
+where:
+
+`<XXXXX>`:: Specifies the random sequence of letters for a pod from the previous step.
+--
++
+.Example output
+[source,text]
+----
+Defaulted container "northd" out of: northd, nbdb, kube-rbac-proxy, sbdb, ovnkube-master, ovn-dbchecker
```

Comment:
probbaly purge this line to aligh with what we have shown in 4.12 and 4.13 changes?